src/modules/tizen/iotjs_module_uart-tizen.c

```diff
@@ -0,0 +1,176 @@
+/* Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <peripheral_io.h>
+
+#include "modules/iotjs_module_uart.h"
+
+struct _peripheral_uart_s {
```

Comment:
This struct seems unused, is it still needed?

Reply:
It needs a forward declaration to refer to `fd` of `peripheral_uart_h`.